docs/rp2/quickref.rst

```diff
@@ -221,7 +230,7 @@ See :ref:`machine.WDT <machine.WDT>`. ::
 Deep-sleep mode
```

Comment:
Lets remove the deep sleep section for now. 

The current implementation uses lightsleep and then resets the device which is not ideal behaviour. Let's fix the implementation first and then update the docs.